### DISCUSSION TITLE
Cisco SPAN > Hyper-V Host > Hyper-V Switch > VM NIC Monitor

### DISCUSSION THREAD

**Discussion by bayden10** [Security-Onion-Solutions/securityonion#2225] [Unsupported Versions] [answered]
Hi,

Anyone have monitoring working on a Hyper-V VM as mentioned in Subject? 

Not many docs out there but I've been able to manage to see traffic from our Core Cisco Switch Trunk to Switch port assigned to a Host NIC where I can capture traffic via WireShark and verify all Trunk traffic is being mirrored. 

Setup a vSwitch which is attached to this NIC on the host. Setup the VM for SO with 2 NICs, second NIC attached to the vSwitch Monitoring port and set the Advanced option to Monitor Destination.

On Windows 2019 DC (Hyper-V Host) I do not see traffic from the Physical NIC to the vSwitch adapter. I seem to be missing a step, set physical NIC as source? and this should allow the vSwitch adapter to receive as Destination?

Enabling Mirroring on the VM 2nd nic as Destination only captures VM to VM traffic, not from the physical NIC as noted within SO Kabana.

Here is a visual
 https://pasteboard.co/JDr6ntw.png

thx
Jim

**Comment by sleepingbel** [comment, marked as answer]
Hello Jim,

Do you use VLAN's? If so you must configure them on the vSwitch.

this is a good article and it uses the same network setup,
[https://cybersecurity.att.com/documentation/usm-anywhere/deployment-guide/hyperv/getting-traffic-from-physical-network-hyper-v.htm](https://cybersecurity.att.com/documentation/usm-anywhere/deployment-guide/hyperv/getting-traffic-from-physical-network-hyper-v.htm)

Regards
Bart

**Reply by bayden10** [reply]
Hi Bart @sleepingbel ,

Thanks for this information, 
One thing that was never noted in all the articles I have read is the bit about disabling the VMQ on the VM NIC that is set to Destination Monitor. 
This article worked for me as yes we also use VLANs and the PowerShell script did not throw an error as many examples have throughout my search. 

Now this is out of the way I seem to have an issue when trying to view _id. Will open another Q&A. about "This job is awaiting completion"

Thx
Jim
